flume-ng-doc/sphinx/FlumeUserGuide.rst

```diff
@@ -2166,10 +2166,31 @@ Property Name            Default  Description
 **kite.dataset.uri**     --       URI of the dataset to open
 kite.repo.uri            --       URI of the repository to open
                                   (deprecated; use kite.dataset.uri instead)
+kite.dataset.namespace   --       Namespace of the Dataset where records will be written
+                                  (deprecated; use kite.dataset.uri instead)
 kite.dataset.name        --       Name of the Dataset where records will be written
                                   (deprecated; use kite.dataset.uri instead)
 kite.batchSize           100      Number of records to process in each batch
 kite.rollInterval        30       Maximum wait time (seconds) before data files are released
+kite.avro.commitOnBatch  true     If true, the writer will be synced and the Flume transaction
```

Comment:
Okay, one more thing on this option. Because we are moving to `Syncable` in the upstream API, it would be better to name this kite.syncable.commitOnBatch so that we can apply it for any Syncable dataset.